server/datastore/mysql/statistics.go

```diff
@@ -84,6 +84,10 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		stats.VulnDetectionEnabled = appConfig.VulnerabilitySettings.DatabasesPath != ""
 		stats.SystemUsersEnabled = appConfig.Features.EnableHostUsers
 		stats.HostsStatusWebHookEnabled = appConfig.WebhookSettings.HostStatusWebhook.Enable
+		stats.MDMMacOsEnabled = appConfig.MDM.EnabledAndConfigured
+		stats.HostExpiryEnabled = appConfig.HostExpirySettings.HostExpiryEnabled
```

Comment:
@lucasmrod just to make sure, is there any chance that 
`appConfig.HostExpirySettings` will null
so  `appConfig.HostExpirySettings.HostExpiryEnabled` will crash ?

Reply:
They are not pointers, so we're good.